android-kotlin/app/src/main/java/com/mongodb/app/RealmApp.kt

```diff
@@ -0,0 +1,37 @@
+package com.mongodb.app
+
+import android.app.Application
+import android.util.Log
+
+import io.realm.Realm
+import io.realm.log.LogLevel
+import io.realm.log.RealmLog
+import io.realm.mongodb.App
+import io.realm.mongodb.AppConfiguration
+
+lateinit var realmApp: App
+
+// global Kotlin extension that resolves to the short version
+// of the name of the current class. Used for labelling logs.
+inline fun <reified T> T.TAG(): String = T::class.java.simpleName
+
+/*
+*  Sets up the Realm App and enables Realm-specific logging in debug mode.
+*/
+class App: Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        Realm.init(this)
+         realmApp = App(
+            AppConfiguration.Builder(getString(R.string.mongodb_realm_app_id))
+                .build())
+
+        // Enable more logging in debug mode
+        if (BuildConfig.DEBUG) {
+            RealmLog.setLevel(LogLevel.ALL)
```

Comment:
This is very verbose and result in a A LOT of log output which might be confusing...for a starter experience I would suggest using `LogLevel.DEBUG`